src/pages/Researchers/ViewVolunteersModal.js

```diff
@@ -1,89 +1,97 @@
-import React, { Fragment, useState } from "react";
-import { Modal, Table} from "antd";
+import React, { Fragment, useState, useEffect } from "react";
+import { Modal, Table } from "antd";
+import Db from "../../util/Database";
 
-const dummyData = {
-  researcher: "Mcflurry",
-  organization: "National University of Pots and Pans",
-  email: "researcher@gmail.com",
-  participants: 15,
-};
+const ViewVolunteersModal = ({ isModalVisible, setModalVisible, item }) => {
+  const [data, setData] = useState(null);
+
+  useEffect(() => {
+    getData();
+  }, []);
+
+  const getData = async () => {
+    const volunteersData = await Db.getUsersData(item.volunteerUserIds);
+    setData(parseVolunteersData(volunteersData));
+  };
```

Comment:
As discussed, getUsersData returns a maximum of 10 users because of the "in" in the WHERE clause of the function. Will commit a workaround for this.